use-cases/using-vpp-and-kata.md

```diff
@@ -0,0 +1,76 @@
+# Setup to run VPP
+
+The Data Plane Development Kit (DPDK) is a set of libraries and drivers for
+fast packet processing. The Vector Packet Processing (VPP) is a platform
+extensible framework which provides out-of-the-box production quality
```

Comment:
sub out "which" for "that": extensible framework that provides out-of-the-box production quality

Reply:
@egernst This needs to be addressed.